packages/genesys-spark-components/src/components/stable/gux-form-field/helper-components/gux-form-field-label-tooltip/gux-form-field-label-tooltip.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, Element, h, JSX, Prop } from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxFormFieldTooltipVariant } from './gux-form-field-label-tooltip.types';
+import { randomHTMLId } from '@utils/dom/random-html-id';
+
+@Component({
+  styleUrl: 'gux-form-field-label-tooltip.scss',
+  tag: 'gux-form-field-label-tooltip'
+})
+export class GuxFormFieldTooltip {
+  private tooltipId: string = randomHTMLId('gux-form-field-label-tooltip');
+
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  variant: GuxFormFieldTooltipVariant = 'info';
+
+  @Prop()
+  screenreaderText: string;
+
+  componentWillLoad(): void {
+    trackComponent(this.root);
+  }
+
+  render(): JSX.Element {
+    return (
+      <button
+        id={this.tooltipId}
+        type="button"
+        class="gux-form-field-label-tooltip"
+      >
+        <gux-icon
+          icon-name={`fa/circle-${this.variant}-regular`}
+          screenreader-text={this.screenreaderText}
+          size="small"
+        ></gux-icon>
+        <gux-tooltip for={this.tooltipId}>
+          <slot></slot>
```

Comment:
Make this a named slot and document it with a comment above `@Component`

Reply:
added a slot called 'content'